docs/command-line/index.mdx

```diff
@@ -136,6 +137,48 @@ testplane --grep "with nested path"
 testplane --grep "test with nested path"
 ```
 
+#### Tag {#testplane-tag}
+
+Run only tests, which matches the tag pattern.
+
+##### Example {#testplane-tag-example}
+
+You can assign tags to a test using the `{tag: string | string[]}` parameter.
+You can also add tags dynamically during test execution with the [addTag][add-tag] command,
+but tests tagged this way cannot be filtered at launch.
+
+```ts
```

Comment:
Missing example of setting multiple tags in one suite/test
There should be example where `tag` is array